dojo/tools/jfrogxray/parser.py

```diff
@@ -59,28 +58,25 @@ def get_item(vulnerability, test):
 
     cve = "No CVE on file"
     cwe = 0
+    cvssv3 = None
```

Comment:
possibly make the variable name more explicit, with regards to what it will truly hold?

Reply:
It will hold the value of the atribute `cvssv3` of the `Finding`. I can rename it to `finding_cvss3` if it make things more clear.

Reply:
I had to go read what the cvss package was doing and the specific function too. So I guess I meant maybe rename to something like `cvss3_clean_vector` ?

Reply:
Forget me, that's the one part of the model :facepalm: